OpenSourceOrg/dotOrg
# [Main] Add/new page template noheader wide main

#### Changes proposed in this Pull Request

* New page template like No Header but wider

#### Testing instructions

* Create a new page
* Assing the template No Page Header - wide
* Confirm the content spans to the width of the breadcrumbs

Mentions #145
